frontend/src/scenes/notebooks/NotebookTemplates/notebookTemplates.ts

```diff
@@ -68,11 +69,56 @@ export const LOCAL_NOTEBOOK_TEMPLATES: NotebookType[] = [
                     ],
                 },
                 { type: 'paragraph', content: [{ type: 'text', text: '- List ' }] },
-                { type: 'paragraph', content: [{ type: 'text', text: '1. Numbered list' }] },
+                {
+                    type: 'bulletList',
```

Comment:
Would this not need to be `orderedList` if we're including numbers?

Reply:
That might be the hook to grab hold of... 

This was just from me typing into the UI. Weirdly the numbered list appears to work correctly but `-` and `*` lists don't

Reply:
I'm switching this up...

Will make a set of "if the JSONContent is correct" stories so we know we render valid JSONContent"

And then we can figure out how to test 

"if someone types `blah` then we would render `<template>blah</template>` or whatevs